eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/apprule/base/EidasSamlParserAppRuleTestBase.java

```diff
@@ -76,7 +75,7 @@ protected Response postEidasAuthnRequest(AuthnRequest authnRequest) throws Marsh
                     .post(Entity.entity(request, MediaType.APPLICATION_JSON_TYPE));
 
         } catch (URISyntaxException e) {
-            fail(e);
+            fail(e.getMessage());
```

Comment:
I don't think we need the `fail`here - the method can `throw` (add to the method signature) this exception and if it does the tests that don't expect the exception will fail naturally.
Also the method will never throw a `MarshallingException ` - this can be removed from the signature.